src/main/jbake/content/blog/2017/07/opentracing-jaxrs.adoc

```diff
@@ -0,0 +1,136 @@
+= OpenTracing JAX-RS Instrumentation
+Pavol Loffay
+2017-7-10
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, apm, tracing, jaeger, opentracing, jax-rs
+
+In the previous http://www.hawkular.org/blog/2017/06/9/opentracing-spring-boot.html[demo]
+we have demonstrated how to instrument Spring Boot app
+using http://opentracing.io[OpenTracing] a vendor-neutral standard for distributed tracing.
+In this article we are going to instrument JAX-RS and show how to trace business layer and add
+custom data to the trace.
+
+== Demo application
+Creating JAX-RS app from scratch might be a time consuming task, therefore in this case we are going to use
+Wildfly Swarm and use http://wildfly-swarm.io/generator/[app generator]! Choose JAX-RS and CDI dependencies and hit generate button.
+
+ifndef::env-github[]
+image::/img/blog/2017/wf-swarm-generator.png[caption="Figure 1: ", title="Wildfly Swarm generator."]
+endif::[]
+ifdef::env-github[]
+image::../../../../../assets/img/blog/2017/wf-swarm-generator.png[caption="Figure 1: ", title="Wildfly Swarm generator."]
+endif::[]
+
+The generated application contains one REST endpoint which returns hello world string.
+This endpoint is accessible on `http://localhost:8080/hello`.
+It is sufficient for our demo and in the next step we are going to add instrumentation and simple business logic.
```

Comment:
I would remove "It is sufficient for our demo and", and have "In the next step ...."